src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -16,154 +17,176 @@ import {
+      {/* Hidden image, used to handle error if image is not loaded */}
+      {/*<img*/}
+      {/*  src={pictureUrl}*/}
+      {/*  onError={(event) => imageErrorHandler(event)}*/}
+      {/*  hidden*/}
+      {/*  style={{ display: "none" }}*/}
+      {/*  alt={user.anonMode ? DEFAULT_PARTY_NAME : user.partyName}*/}
+      {/*/>*/}
+
+      {/* TODO: can we use src/components/atoms/UserAvatar/UserAvatar.tsx here? Should we? */}
+      <div
+        role="img"
+        aria-label={`${userDisplayName}'s avatar`}
+        className={avatarClasses}
+        style={avatarStyles}
+        onClick={openProfileModal}
+      />
+
+      {/* TODO: refactor this to be more performant and potentially memo it if required */}
+      <div className="UserProfilePicture__reaction-container">
         {Reactions.map(
           (reaction, index) =>
             reactions.find(
               (r) => r.created_by === user.id && r.reaction === reaction.type
             ) && (
-              <div key={index} className="reaction-container">
-                <S.Reaction
+              // TODO: don't use index as a key
+              // TODO: should we extract reactions into their own sub-components?
+              // TODO: is reaction-container even defined in our styles here..?
+              <React.Fragment key={index}>
+                <div
+                  className="UserProfilePicture__reaction"
                   role="img"
                   aria-label={reaction.ariaLabel}
-                  className={reaction.name}
-                  reactionPosition={reactionPosition}
                 >
                   {reaction.text}
-                </S.Reaction>
+                </div>
 
+                {/* TODO: replace this with useSound or similar */}
                 {!muteReactions && !isAudioEffectDisabled && (
                   <audio autoPlay loop>
                     <source src={reaction.audioPath} />
                   </audio>
                 )}
-              </div>
+              </React.Fragment>
             )
         )}
+
         {messagesToBand && (
-          <div className="reaction-container">
-            <S.ShoutOutMessage
-              role="img"
-              aria-label="messageToTheBand"
-              reactionPosition={reactionPosition}
-            >
-              {messagesToBand.text}
-            </S.ShoutOutMessage>
-          </div>
+          <div className="UserProfilePicture__shout">{messagesToBand.text}</div>
         )}
-      </S.Container>
-    );
-  }, [
-    containerStyle,
-    pictureUrl,
-    user,
-    avatarClassName,
-    avatarStyle,
-    messagesToBand,
-    reactionPosition,
-    imageErrorHandler,
-    openUserProfileModal,
-    reactions,
-    muteReactions,
-    isAudioEffectDisabled,
-  ]);
+      </div>
+    </div>
+  );
 };
 
+/**
+ * @deprecated Prefer using named exports instead of default exports
+ */
 export default UserProfilePicture;
```

Comment:
[TODO] Remove this + update all locations across the app that use it.

Reply:
It looks like this is only used in ~5 files currently.. so wouldn't create too much noise to do it.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1323/commits/14f3867baa069138b1e6510c45f6e09f9f08b7d4